examples/nuxt-2-composition-api/README.md

```diff
@@ -0,0 +1,68 @@
+# nuxt-starter
+
+## Build Setup
```

Comment:
Since this is an example instead of a starter, it might be worth adding a bit more of a description to the Readme. Giving a little background on the example, and what users might want to get from it. Possibly how to clone it as a starter if they want to do so.